__mocks__/react-native-mmkv.ts

```
@@ -10,7 +10,18 @@ class MockMMKV {
   private prefix: string;
 
   constructor(config?: { id?: string; encryptionKey?: string }) {
-    this.storage = typeof window !== 'undefined' ? window.localStorage : ({} as Storage);
+    const _fallbackData: Record<string, string> = {};
+    const _fallbackStorage: Storage = {
+      getItem(key: string) { return _fallbackData[key] ?? null; },
+      setItem(key: string, value: string) { _fallbackData[key] = value; },
+      removeItem(key: string) { delete _fallbackData[key]; },
+      key(index: number) { return Object.keys(_fallbackData)[index] ?? null; },
+      get length() { return Object.keys(_fallbackData).length; },
+      clear() { Object.keys(_fallbackData).forEach((k) => delete _fallbackData[k]); },
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)